cmd/contour/contour.go

```diff
@@ -45,7 +45,7 @@ func main() {
 
 	// Log-format applies to log format of all sub-commands.
 	logFormat := app.Flag("log-format", "Log output format for Contour. Either text or json.").Default("text").Enum("text", "json")
-
+	logPath := app.Flag("log-path", "Log output path.").Default("").String()
```

Comment:
The description could mention also that when path is not defined, the logs are printed in stderr.

Reply:
Yeah makes sense, done